packages/next-app/next.config.js

```diff
@@ -1,7 +1,10 @@
 /** @type {import('next').NextConfig} */
 
 module.exports = {
-  transpilePackages: ["@balancer-pool-metadata/balancer-gql"],
+  transpilePackages: [
+    "@balancer-pool-metadata/balancer-gql",
+    "@balancer-pool-metadata/pool-metadata-gql",
```

Comment:
`#comment` so much to write 🥵 starting to regret these names. But I don't have any better suggestions either 😅